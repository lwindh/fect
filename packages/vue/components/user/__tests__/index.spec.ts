import { User } from '..'
import { mount } from '@vue/test-utils'

describe('User', () => {
  it('should render as normal', () => {
    const wrapper = mount(User, {
      props: {
        text: 'Kanno',
        name: 'kanno'
      }
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
