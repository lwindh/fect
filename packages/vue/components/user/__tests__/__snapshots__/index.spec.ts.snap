// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User should render as normal 1`] = `
<div class="fect-user">
  <div class="fect-avatar small ">
    <!----><span class="fect-avatar-text">Kan</span>
  </div>
  <div class="fect-user__names"><span class="name">kanno</span><span class="social"><!----></span></div>
</div>
`;
